project_script/main.py

```diff
@@ -82,19 +83,16 @@ def integrate_data():
 
 sanitize_data_frames()
 
-data_ramp_time = pd.concat(
-        [time_photo_celiv_ramp_time, odd_columns_subtracted],
-        axis=1
-    )\
+data_ramp_time = pd.concat([time_photo_celiv_ramp_time, odd_columns_subtracted], axis=1)\
     .sort_index(axis=1)
 
 # data_ramp_time.to_excel("C:/Users/robee/Desktop/data_ramp_time.xlsx")
 
-data_ramp_time.to_csv(ask.file_name(), sep='\t')     # line commented to run tests
+# data_ramp_time.to_csv(ask.file_name(), sep='\t')     # line commented to run tests
 
 data_ramp_time_transposed_in_arrays = data_ramp_time.transpose().to_numpy()
 
-time_photo_celiv_ramp_time_transposed = time_photo_celiv_ramp_time.transpose().to_numpy()
+time_photo_celiv_ramp_time_transposed = time_photo_celiv_ramp_time.dropna().transpose().to_numpy()
```

Comment:
In particular, i would break in this case with more than three constraint. Ps. can't remember if the right name in this case is `constraint` or `chaining`. I guess constraint it's something similar but used in construction of the database 
`queries`, and in this case we're  just `chaining the methods`.

```
time_photo_celiv_ramp_time_transposed = time_photo_celiv_ramp_time.dropna()
    .transpose()
    .to_numpy()
```